### PR TITLE
182448 - 182449 - open api specs stub and code changes

### PR DESCRIPTION
**Pull request by chaital14503**
This PR includes code changes for [PBI-182448](https://dev.azure.com/ukhydro/S-100%20Permit%20Generator/_workitems/edit/182448)  and  [PBI-182449](https://dev.azure.com/ukhydro/S-100%20Permit%20Generator/_workitems/edit/182449) 

Code and stub changes, UTs, FTs

### CODE REVIEW

File: tests/UKHO.S100PermitService.Common.UnitTests/Services/PermitServiceTests.cs [hunk 1]
```diff
@@ -261,106 +280,185 @@ public async Task WhenHoldingsServiceReturnsNotFoundResponse_ThenPermitServiceRe
...
+                    return (new HttpResponseMessage()
+                    {
+                        StatusCode = HttpStatusCode.OK
+                    },
                     [
                         new HoldingsServiceResponse
                         {
-                            ProductTitle = "ProductTitle",
-                            ProductCode = "ProductCode",
+                            UnitTitle = "ProductTitle",
```
Comment (harshal11869) [inline tests/UKHO.S100PermitService.Common.UnitTests/Services/PermitServiceTests.cs:328]: HoldingServiceResponse fields should be in Camel Case e.g. unitTitle, unitName please check all responses in this class.

**Review by harshal11869 — CHANGES_REQUESTED** [review]

File: src/UKHO.S100PermitService.Common/Services/HoldingsService.cs [hunk 2]
```diff
@@ -73,61 +73,65 @@ public HoldingsService(ILogger<HoldingsService> logger, IOptions<HoldingsService
         /// <returns>Filtered holding details.</returns>
         public IEnumerable<HoldingsServiceResponse> FilterHoldingsByLatestExpiry(IEnumerable<HoldingsServiceResponse> holdingsServiceResponse)
         {
-            var allCells = holdingsServiceResponse.SelectMany(p => p.Cells.Select(c => new { p.ProductCode, p.ProductTitle, p.ExpiryDate, Cell = c }));
+            var allCells = holdingsServiceResponse.SelectMany(p => p.Datasets.Select(c => new { p.UnitName, p.UnitTitle, p.ExpiryDate, Cell = c }));
```
[the remark this answers is not in the document]
Comment (mayur1501364) [reply src/UKHO.S100PermitService.Common/Services/HoldingsService.cs:76]: Renamed

File: src/UKHO.S100PermitService.Common/Events/EventIds.cs [hunk 3]
```diff
@@ -91,153 +91,163 @@ public enum EventIds
...
+
+        /// <summary>
+        /// 840048 - Request to UserPermitService GetUserPermits responded licence not found
+        /// </summary>
+        UserPermitServiceGetUserPermitsLicenceNotFound = 840048,
+
+        /// <summary>
+        /// 840049 - Request to HoldingsService GetHoldings responded licence not found
+        /// </summary>
+        HoldingServiceGetHoldingsLicenceNotFound = 840049
```
[the remark this answers is not in the document]
Comment (mayur1501364) [reply src/UKHO.S100PermitService.Common/Events/EventIds.cs:251]: Renamed event

File: tests/UKHO.S100PermitService.Common.UnitTests/Services/PermitServiceTests.cs (hunk 1 above)
Comment (mayur1501364) [reply tests/UKHO.S100PermitService.Common.UnitTests/Services/PermitServiceTests.cs:328]: Resolved

**Comment by shirinbt14926** [comment]
Closing this PR as new PR will be raised for Open API spec with Result Pattern

**Pull request closed without merging**